Signal/src/call/CallService.swift

```diff
@@ -74,7 +74,8 @@ enum CallError: Error {
     case assertionError(description: String)
     case disconnected
     case externalError(underlyingError: Error)
-    case timeout(description: String, call: SignalCall)
```

Comment:
When call2 immediately follows call1, we don't want to accidentally cancel call2 when (the already ended) call1 times out.